datalake/src/main/java/com/sequenceiq/datalake/flow/stop/SdxStopActions.java

```diff
@@ -191,6 +192,8 @@ protected void doExecute(SdxContext context, SdxFailedEvent payload, Map<Object,
                 if (exception.getMessage() != null) {
                     statusReason = exception.getMessage();
                 }
+                Flow flow = getFlow(context.getFlowParameters().getFlowId());
+                flow.setFlowFailed(payload.getException());
```

Comment:
same here

Reply:
This piece of code is in "SDX_STOP_FAILED_STATE" state. When the flow reaches this state, it is clear that the stop of SDX failed. I do not understand why you think stop flow enters this state without an actual failure?

Reply:
same here, I haven't seen it is a FAILED state, then it should be ok, sorry